### ISSUE TITLE
Unit test failure stops build, unit test method testcreateContestStandings 

### ISSUE THREAD

**Issue by lane55** [pc2ccs/pc2v9#279]
**Describe the issue**: 

Unit test is failing.  ScoreboardJsonTest.testcreateContestStandings fails.

java.io.FileNotFoundException under Unix.
Change \ in unit test filename to / to fix.

**To Reproduce**: 

Run the JUnit under Unix/Linux.

**Expected behavior**: 

Unit test during build ScoreboardJsonTest.testcreateContestStandings passes.

**Actual behavior**: 

Unit test during build ScoreboardJsonTest.testcreateContestStandings fails.

From gitlab build 

Testcase:
testcreateContestStandings(edu.csus.ecs.pc2.services.core.ScoreboardJsonTest):
Caused an ERROR
[junit] null
[junit] javax.xml.bind.UnmarshalException
[junit] - with linked exception:
[junit] [java.io.FileNotFoundException:
/builds/pc2ccs/pc2v9/testdata\ScoreboardJSONTest\testNADC21Prac2EF\nadcPractice-event-feed.xml
(No such file or directory)]

**Environment**: 

**Log Info**:  

**Screenshots**:  

**Additional context**:

**Comment by clevengr** [comment]
I approved PR #280, merged it, and examined the resulting build -- which failed.

The slash-vs-backslash MAY `also` have caused a failure, but that's not the (only) reason for the failure.  I just discovered the following:
- the test data file is named in the repository as `testdata/ScoreboardJsonTest/testNADC21Prac2EF/nadcPractice-event-feed.xml`.
- the path to the test data is declared in the JUnit code as
``` java
String xmlFilename = "testdata/ScoreboardJSONTest/testNADC21Prac2EF/nadcPractice-event-feed.xml";
```

Note the different case in the folder names:  `ScoreboardJsonTest` vs. `ScoreboardJSONTest`  ("Json" vs. "JSON").  This works under Windows, but different case in path names doesn't work under Unix.
